modules/moa-installing-prerequisites.adoc

```diff
@@ -25,6 +25,10 @@ This user will have Programmatic access enabled and the AdministratorAccess poli
 +
 . Install and configure the AWS CLI.
 .. Follow the AWS documentation to install the link:https://aws.amazon.com/cli/[aws-cli] for your operating system.
+.. `moactl` handles AWS regions in this priority order:
```

Comment:
I'm not sure where this information should go, but it should be a note somewhere else instead of a step here - there's no explicit action.